codecoverage.sh

```diff
@@ -11,3 +11,4 @@ fi
 lcov --quiet -t 'OOC Test coverage' -o $INFO_FILE -c --directory ./.libs/ooc/ --base-directory .
 genhtml --quiet -o coverage $INFO_FILE --num-spaces 4
 rm $INFO_FILE
+xdg-open 'coverage/index.html'
```

Comment:
xdg-open is very standardized but maybe I check like this would be good?
if which xdg-open > /dev/null
then
  xdg-open 'coverage/index.html'
elif which gnome-open > /dev/null
then
  gnome-open 'coverage/index.html'
else
  echo "Could not detect the web browser to use."
fi